homeassistant/components/light/services.yaml

```diff
@@ -574,3 +574,17 @@ toggle:
       description: Light effect.
       selector:
         text:
+brightness_increase:
+  name: Brightness Increase
+  description: >
+    Increase the brightness by 10%
+  target:
+    entity:
+      domain: light
+brightness_decrease:
+  name: Brightness Decrease
+  description: >
+    Decrease the brightness by 10%
+  target:
+    entity:
+      domain: light
```

Comment:
These services do not seem to exists, what is the reason you are adding them to the service descriptions?

Reply:
Ah ok. I'm probably being dense then. I'm trying to learn how all this glues together and I probably confused the too.


Sorry; Ignore for now.